client/aws/aws.go

```diff
@@ -207,7 +208,7 @@ func fillInstancePayload(instanceRecord instanceRecord, offerList []*client.Offe
 			Type:               entry.Attributes.Type,
 			InstanceFamily:     entry.Attributes.InstanceFamily,
 			CPU:                entry.Attributes.CPU,
-			Memory:             entry.Attributes.Memory,
+			Memory:             entry.Attributes.Memory[:strings.Index(entry.Attributes.Memory, "GiB")-1],
```

Comment:
It's quite brittle do index of a string, e.g. what if string doesn't have GiB or starts with GiB? Recommend to use strings.ReplaceAll if Memory always ends with GiB.